cpp/src/io/orc/stripe_enc.cu

```diff
@@ -44,7 +45,11 @@ namespace io {
 namespace orc {
 namespace gpu {
 
+namespace nvcomp = cudf::io::detail::nvcomp;
```

Comment:
I've thought about this (briefly :D ), and went ahead because I've never seen a namespace in nvcomp; the header files we include are `extern "C" {` in the global namespace. Also, no file outside of the adapter should include anything from nvcomp. 
FWIW, these files should stop depending on the nvcomp adapter (and thus the namespace) once I implement the high-level API that uses nvcomp transparently.

Reply:
all that said, I should have used a different name :D

Reply:
I checked that `nvcomp.hpp` has `namespace nvcomp`, but I did not look at whether it has `extern C`. Let’s not worry unless it presents problems.